### PR TITLE
move Vanadium configuration information on build page

### PR DESCRIPTION
**Pull request by theaeonsolution**

### CODE REVIEW

File: static/build.html [hunk 1]
```diff
@@ -656,21 +656,18 @@ <h3 id="browser-and-webview">
...
 
-            <p>You can obtain the proper configuration from the
-            <a href="https://github.com/GrapheneOS/Vanadium">
-            GrapheneOS Vanadium repository</a> in <code>args.gn</code> including the correct
-            version.</p>
-
             <pre>git clone https://github.com/GrapheneOS/Vanadium.git
-cd Vanadium
-git checkout $CORRECT_BRANCH_OR_TAG</pre>
+            cd Vanadium
```
Comment (thestinger) [inline static/build.html]: This whitespace shouldn't be added since the pre tag will include it.

**Review by thestinger — CHANGES_REQUESTED** [review]

File: static/build.html (hunk 1 above)
Comment (theaeonsolution) [reply static/build.html]: Update made. I really thought I could squash commits on the site, I will try to make less noise with changes and work locally before pushing commits.

File: static/build.html [hunk 2]
```diff
@@ -669,8 +664,10 @@ <h3 id="browser-and-webview">
 
             <pre>fetch --nohooks android</pre>
 
-            <p>Sync to the latest stable release for Android (replace $VERSION with the correct
-            value):</p>
+            <p>You can obtain the proper configuration from the <a href="https://github.com/GrapheneOS/Vanadium">
```
Comment (thestinger) [inline static/build.html:667]: You should probably move this down a bit further if you want to move it like this. I don't think it makes more sense here. It should go with the configuration step.

**Review by thestinger — CHANGES_REQUESTED** [review]

File: static/build.html (hunk 2 above)
Comment (theaeonsolution) [reply static/build.html:667]: Thanks, let me review.

**Pull request closed without merging**